### PR TITLE
Ser 3281 sentry integration

### PR DESCRIPTION
**Pull request by gilclefes**
## Description
Integrate Fineract UI with Sentry for error analytics and tracing

## Changes Included in PR
-https://oneacrefund.atlassian.net/browse/SER-3281

## PR Checklist
- [ ] Xray tests created
- [ ] PM tested and approved
- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`

## Screenshots, if any

.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -26,12 +26,7 @@
             "main": "src/main.ts",
             "tsConfig": "src/tsconfig.app.json",
             "polyfills": "src/polyfills.ts",
-            "assets": [
```
Comment (amydoxym) [inline angular.json:29]: @gilclefes It's hard to see which lines was modified in this file...can you just rework on this and commit only lines changed

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -1,41 +1,30 @@
 <div fxLayput="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
-
```
Comment (amydoxym) [inline src/app/home/home.component.html:2]: same comment.. which line was modified here?

File: src/environments/environment.test.ts [hunk 3]
```diff
@@ -1,39 +1,40 @@
 // `.env.ts` is generated by the `npm run env` command
-import env from "./.env";
```
Comment (amydoxym) [inline src/environments/environment.test.ts:2]: revert change from " to '

**Review by amydoxym — CHANGES_REQUESTED** [review]

File: angular.json (hunk 1 above)
Comment (gilclefes) [reply angular.json:29]: for this the this was the only change  **"sourceMap": true** the rest are automatic formating

File: src/app/home/home.component.html (hunk 2 above)
Comment (gilclefes) [reply src/app/home/home.component.html:2]: No file actually got modified here, i added a button to test and event and after removing it , some automatic formatting was applied

File: src/environments/environment.test.ts (hunk 3 above)
Comment (gilclefes) [reply src/environments/environment.test.ts:2]: @amydoxym i guess we maintain ', as the other environment files are using same.

**Review by amydoxym — APPROVED** [review]

**Review by victorjbassey — APPROVED** [review]

**Pull request merged**